src/rred_reports/reports/schools.py

```diff
@@ -70,51 +70,54 @@
 
 
 def school_filter(whole_dataframe: pd.DataFrame, school_id):
-    """Function to filter school"""
-    return whole_dataframe[whole_dataframe.school_id == school_id].copy()
+    """Function to filter out the Teacher Leaders and by school
 
+    Args: pd.DataFrame, school_id
 
-# specific filter for each table
-def filter_one(school_dataframe: pd.DataFrame):
-    """Filter for table one, for later use
-
-    Args: school_filter(pd.DataFrame)"""
-    return school_dataframe
-
-
-def filter_two(school_dataframe: pd.DataFrame):
-    """Filter for table two, for later use
-
-    Args: school_filter(pd.DataFrame)"""
-    return school_dataframe
+    Returns: pd.DataFrame filtered to show only RR teachers data per school
 
+    """
+    teacher_filtered = whole_dataframe[
+        (whole_dataframe.reg_rr_title == "RR Teacher + Support Role") | (whole_dataframe.reg_rr_title == "RR Teacher + Class Leader")
+    ]
+    return teacher_filtered[teacher_filtered.school_id == school_id].copy()
 
-def filter_three(school_dataframe: pd.DataFrame):
-    """Filter for table three, for later use
 
-    Args: school_filter(pd.DataFrame)"""
-    return school_dataframe
+# specific filter for each table
+def filter_for_one_two_five(school_dataframe: pd.DataFrame):

```

Comment:
all of the filter functions should take a year and then we do the dates based on that year! The top level function will also need to have a year passed in too